go/flags/endtoend/vtorc.txt

```diff
@@ -32,6 +32,7 @@ Flags:
       --config-persistence-min-interval duration                    minimum interval between persisting dynamic config changes back to disk (if no change has occurred, nothing is done). (default 1s)
       --config-type string                                          Config file type (omit to infer config type from file extension).
       --consul_auth_static_file string                              JSON File to read the topos/tokens from.
+      --discovery-max-concurrency int                               Number of workers used for tablet discovery (default 300)
```

Comment:
This is somewhat of a nit, but this isn't really a "max" is it? It's the actual concurrency. We are creating as many goroutines as this number tells us to.

Reply:
@deepthi good catch! I think "concurrency" is also misleading because all workers aren't guaranteed to be doing any work

I've renamed this to `--discovery-workers` which I think tells the story better. Please merge if you agree